### PR TITLE
Implement all members routes

### PR DESCRIPTION
**Pull request by jay3332**
Includes ``get_member``, ``delete_member``, and ``create_member``.

### CODE REVIEW

File: ferrischat_webserver/src/members/delete_member.rs [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+        u128_to_bigdecimal!(raw)
+    };
+    let member_id = {
+        let raw = get_item_id!(req, "member_id");
+        u128_to_bigdecimal!(raw)
+    };
+    
+    let db = get_db_or_fail!();
+
+    let resp = sqlx::query!("DELETE FROM members WHERE id = $1 AND guild_id = $2 RETURNING (id)", member_id, guild_id)
```
Comment (hydrostaticcog) [inline ferrischat_webserver/src/members/delete_member.rs:17]: the member object does not have its own ID. use the user's ID.

File: ferrischat_webserver/src/members/create_member.rs [hunk 2]
```diff
@@ -1,6 +1,36 @@
-use actix_web::{HttpResponse, Responder};
```
Comment (hydrostaticcog) [inline ferrischat_webserver/src/members/create_member.rs:1]: i don't like the idea of being able to join a guild via an endpoint. that is what the invite system is for.

**Review by hydrostaticcog — CHANGES_REQUESTED** [review]

**Comment by hydrostaticcog** [comment]
also `cargo fmt ` please

File: ferrischat_webserver/src/members/create_member.rs (hunk 2 above)
Comment (tazz4843) [reply ferrischat_webserver/src/members/create_member.rs:1]: +1

I never got around to actually deleting the endpoint, so if you want you can do it now.

**Review by tazz4843 — COMMENTED** [review]
Looks good besides create member.

**Comment by jay3332** [comment]
Ran `cargo fmt` and fix changes.

**Review by tazz4843 — CHANGES_REQUESTED** [review]
Remove the mod from the mod.rs file and the endpoint from entrypoint.rs too.

File: ferrischat_webserver/src/entrypoint.rs [hunk 3]
```diff
@@ -110,17 +110,12 @@ pub async fn entrypoint() {
             // GET    guilds/{guild_id}/members/{member_id}
             .route(
                 expand_version!("guilds/{guild_id}/members/{member_id}"),
```
Comment (tazz4843) [inline ferrischat_webserver/src/entrypoint.rs:112]: You need to remove the entire route() fn call for it to actually be removed. Right now it'll just compile error.

**Review by tazz4843 — CHANGES_REQUESTED** [review]

File: ferrischat_webserver/src/entrypoint.rs (hunk 3 above)
Comment (hydrostaticcog) [reply ferrischat_webserver/src/entrypoint.rs:112]: AKA delete lines 105-109

**Review by hydrostaticcog — CHANGES_REQUESTED** [review]
completely remove create member from entrypoint.rs, then it looks good.

**Comment by jay3332** [comment]
Should be fixed now

**Review by tazz4843 — DISMISSED** [review]

**Comment by jay3332** [comment]
test

**Review by hydrostaticcog — APPROVED** [review]
looks good, not tested though. I approve for merge though.

**Review by hydrostaticcog — COMMENTED** [review]
Looks good, but needs to build.

**Review by tazz4843 — DISMISSED** [review]
Builds successfully against a test DB.

**Review by tazz4843 — APPROVED** [review]

**Pull request merged**